templates/shared/forms/form-fields.html

```diff
@@ -0,0 +1,208 @@
+<section class="p-section">
+  <form {% if isModal %}class="js-modal-form"{% endif %}
+        action="/marketo/submit"
+        method="post"
+        id="mktoForm_{{ formData.formId }}"
+        onsubmit="getCustomFields(event)">
+    {% for fieldset in fieldsets %}
+      <div class="p-section">
+        <hr class="is-fixed-width" />
+        <fieldset class="p-fieldset-section {% if fieldset.id=='ubuntu-versions' %} js-toggle-checkbox-visibility {% elif fieldset.isRequired and fieldset.fields[0].type=='checkbox' %} js-required-checkbox {% endif %}"
+                  id="{{ fieldset.id }}"
+                  aria-labelledby="{{ fieldset.id }}"
+                  style="border: none;">
+          <div class="row--50-50 {% if not fieldset.noCommentsFromLead %}js-formfield{% endif %}">
+            <div class="col">
+              <legend class="p-heading--2 js-formfield-title {% if fieldset.isRequired %}is-required{% endif %}"
+                      id="{{ fieldset.id }}-legend">{{ fieldset.title }}</legend>
+            </div>
+            <div class="col">
+              <ul class="p-list">
+                {% for field in fieldset.fields %}
+                  {% if field.type == "country" %}
+                    {% include "shared/forms/_country.html" %}
+                  {% else %}
+                    <li class="p-list__item">
+                      {% if field.type == "text" or field.type == "tel" or field.type == "email" %}
+                        <label {% if field.isRequired %}class="is-required"{% endif %}
+                               for="{{ field.id }}">
+                          {{ field.label
+                          }}:
+                        </label>
+                        <input {% if field.isRequired %}required{% endif %}
+                               id="{{ field.id }}"
+                               name="{{ field.id }}"
+                               maxlength="255"
+                               type="{{ field.type }}"
+                               {% if field.type=="email" %}pattern="^[^ ]+@[^ ]+\.[a-z]{2,26}$"{% endif %} />
+                      {% elif field.type == "long-text" %}
+                        <textarea {% if fieldset.isRequired %}required{% endif %}
+                                  id="{{field.id}}"
+                                  rows="5"
+                                  maxlength="2000"></textarea>
+                      {% elif field.type == "radio" %}
+                        <label class="p-radio">
+                          <input {% if fieldset.isRequired %}required{% endif %}
+                                 class="p-radio__input"
+                                 type="radio"
+                                 aria-labelledby="{{ field.id }}"
+                                 name="{{ fieldset.inputName }}"
```

Comment:
This will always add radio fieldset value to the payload rather than in Comments_from_lead__C. Can we do a check rather than explicitly setting the name here? To allow for radio input values that should be added to Comments rather than payload.

Reply:
We can just avoid setting input name on the field to not add it to the payload. This is an easier way of handling it then additional checks